base/App/Profile/RegisterScreen.swift

```diff
@@ -17,19 +17,10 @@
 import SwiftUI
 
 struct RegisterScreen: View {
-    @ObservedObject var registerViewModel: RegisterViewModel
+    @StateObject var registerViewModel: RegisterViewModel
```

Comment:
This makes sense in the context of this view is in the TabView... but curious if these should be ObservedObject and their parent view should be the StateObject.. 🤔 

I try not to use StateObjects unless absolutely needed.. We could use it here though I think.